bees/devrantbee/devrantbeefactory.go

```diff
@@ -74,52 +74,57 @@ func (factory *DevrantBeeFactory) Events() []bees.EventDescriptor {
 			Description: "is triggered after rants were fetched",
 			Options: []bees.PlaceholderDescriptor{
 				{
-					Name:        "ID",
+					Name:        "Id",
```

Comment:
Renaming here _was_ necessary because we're parsing the Placeholders name in devrantbee.go.
The returned names are the same as in the particular struct fields (in this case: [devrant](https://godoc.org/github.com/jayeshsolanki93/devgorant#RantModel)).

A simple solution should be to write/find a go library which parses 'camelCase' strings into 'snake_cases'. Therefore we could convert the structs field name just before we translate them as bees.Placeholder name and stay at json styled 'snake_cases' for our bees.Placeholder and bees.PlaceholderDescriptor.